problems/objects.js

```diff
@@ -215,7 +333,15 @@ const mostCommonElement = () => {};
  * @returns {string[]} Elements or their pair values.
  */
 
-const updateList = () => {};
+const updateList = (pairs, arr) => {
+  for(let i = 0; i<arr.length;i++){
+    val = arr [i]
```

Comment:
make sure that the bracket notation is next to the array variable, extensions like prettier code formatter solves these type of problems